Linshuo/server/src/server.cpp

```diff
@@ -0,0 +1,63 @@
+#include <ros/ros.h>
+#include <client/time.h>
+#include <client/show.h>
+
+#include <vector>
+#include <iostream>
+
+client::show srv;
+std::vector<ros::Subscriber> server_subscribers;
+ros::ServiceServer service_server;
+
+void time_callback(const client::time::ConstPtr& time)
+{
+    // 将接收到的消息打印出来
+    ROS_INFO("client_name:%s   time:%lu.%lu.%lu %lu:%lu:%lu ", time->name.c_str(), time->year, time->month, time->day, time->hour, time->minute, time->second);

```

Comment:
**[cpplint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br> Lines should be <= 80 characters long  [whitespace/line_length] [2]